File: 1 - BEGINNER/Java/1013 - O Maior.java
import java.io.IOException;
import java.util.Scanner;
import java.lang.Math;

public class Main {
 
    public static void main(String[] args) throws IOException {
 
        Scanner input = new Scanner(System.in);
        
        int a = input.nextInt();
        int b = input.nextInt();
        int c = input.nextInt();
        
        int maiorAB = (a + b + Math.abs(a-b)) / 2;
        int maiorABC = (maiorAB + c + Math.abs(maiorAB - c)) / 2;
        
        System.out.printf("%d eh o maior\n", maiorABC);

        input.close();
        
    }
 
}

File: 1 - BEGINNER/C++/1013 - O Maior.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    
    int a, b, c, maiorAB, maiorABC;
    
    cin >> a >> b >> c;
    
    maiorAB = (a + b + abs(a - b)) / 2.0;
    maiorABC = (maiorAB + c + abs(c - maiorAB)) / 2.0;
    
    cout << maiorABC << " eh o maior" << endl;
    
    return 0;
}